screens/IntroAppScreen.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {
+  Image,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  TouchableHighlight
+} from 'react-native';
+import { WebBrowser } from 'expo';
+import HomeScreen from './HomeScreen';
+import CreateAccScreen from './CreateAccScreen';
+import { MonoText } from '../components/StyledText';
+
+export default class IntroAppScreen extends React.Component {
+  static navigationOptions = {
+    header: null,
+  };
+
+  render() {
+    return (    	
+      <View style={styles.container}>
+        <View style={[{ position: 'absolute', top: 0 }]}>
+          <Image
+            source={
+              __DEV__
+                ? require('../assets/images/logo.png')
+                : require('../assets/images/logo.png')
+            }
+            style={styles.welcomeImage}
+          />
+        </View>
+        <Text style={styles.welcomeText}>Simplify</Text>
+        <Text style={styles.welcomeText}>Your</Text>
+        <Text style={styles.welcomeText}>Selling</Text>
+
+        <View style={[{ width: "100%", backgroundColor: "#F39F86", position: 'absolute', bottom: 56, borderRadius: 4, }]}>
+  				<TouchableHighlight onPress={() => this.props.navigation.navigate('CreateAcc')}>
```

Comment:
Code formatting looks to be all over the place. Please apply appropriate formatting across the project.